tests/tck/features/go/Orderby.feature

```diff
@@ -22,7 +22,12 @@ Feature: Orderby Sentence
       """
       ORDER BY $-.xx
       """
-    Then a SyntaxError should be raised at runtime:
+    Then a SemanticError should be raised at runtime: `$-.xx', not exist prop `xx'
+    When executing query:
+      """
+      ORDER BY 1
+      """
+    Then a SemanticError should be raised at runtime: Order by with invalid expression `1'
```

Comment:
I think you could let order by only accept VariablePropExpr and InputPropExpr in parser.

Reply:
Order by constants is just ok.

Reply:
> I think you could let order by only accept VariablePropExpr and InputPropExpr in parser.

Currently `Match_order_by` and `Go order by` share some data structures in parser